### DISCUSSION TITLE
希望支持鼠标相对移动

### DISCUSSION THREAD

**Discussion by wh201906** [taojy123/KeymouseGo#185] [Ideas]
RT，目前仅支持移动到屏幕上的某个坐标点，不支持相对当前鼠标位置移动一段距离

**Comment by wh201906** [comment]
从接口上来看不设置win32con.MOUSEEVENTF_ABSOLUTE这个flag就可以了

**Comment by ZutJoe** [comment]
如果你是想自己编写脚本，我认为可以通过知道最开始坐标的位置通过位移来推算出来下一个的绝对位置。以此来实现你的想法

**Reply by wh201906** [reply]
请问怎么拿到脚本启动时候鼠标的位置呢？

**Reply by ZutJoe** [reply]
可能是我自己想不到类似的用途，抱歉

可以的话会增加相关功能

**Reply by wh201906** [reply]
> 可能是我自己想不到类似的用途，抱歉

有些时候使用者可能会需要手动干预，比如用鼠标指在某个按钮上再运行脚本，而按钮出现的位置不固定

**Reply by ZutJoe** [reply]
> 有些时候使用者可能会需要手动干预，比如用鼠标指在某个按钮上再运行脚本，而按钮出现的位置不固定

好的，已经创建相关issue了，谢谢您的建议

**Reply by wh201906** [reply]
谢谢
